### PR TITLE
add update postcard information feature

### PR DESCRIPTION
**Pull request by IraMatsuk**

### CODE REVIEW

File: src/main/java/by/matsuk/shop/controller/command/impl/admin/GoToUpdateProductPageCommand.java [hunk 1]
```diff
@@ -34,6 +34,7 @@ public Router execute(HttpServletRequest request) throws CommandException {
                 return router;
             }
             request.setAttribute(PRODUCT_MENU, catalog.get());
```
Comment (IlyaMatsuk) [inline src/main/java/by/matsuk/shop/controller/command/impl/admin/GoToUpdateProductPageCommand.java]:
```suggestion
            if (! catalog.isEmpty()) {
                request.setAttribute(PRODUCT_MENU, catalog.get());
                request.setAttribute(PRODUCT_ID, postcardId);
                router.setCurrentPage(UPDATE_PRODUCT_PAGE);
            } else {
                router.setRedirectType();
                router.setCurrentPage(ERROR_500);
            }
```

**Review by IlyaMatsuk — CHANGES_REQUESTED** [review]

File: src/main/java/by/matsuk/shop/controller/command/impl/admin/InsertNewProductCommand.java [hunk 2]
```diff
@@ -24,12 +24,12 @@ public class InsertNewProductCommand implements Command {
     @Override
     public Router execute(HttpServletRequest request) throws CommandException {
         Router router = new Router();
-        Map<String,String> map = new HashMap<>();
+        Map<String, String> map = new HashMap<>();
```
Comment (IlyaMatsuk) [inline src/main/java/by/matsuk/shop/controller/command/impl/admin/InsertNewProductCommand.java]: please rename `map`

**Review by IlyaMatsuk — CHANGES_REQUESTED** [review]

File: src/main/java/by/matsuk/shop/controller/command/impl/admin/UpdateProductCommand.java [hunk 3]
```diff
@@ -21,26 +19,25 @@
...
-    private static final Logger logger = LogManager.getLogger();
     private final MenuServiceImpl service = MenuServiceImpl.getInstance();
 
     @Override
     public Router execute(HttpServletRequest request) throws CommandException {
         Router router = new Router();
-        Map<String,String> map = new HashMap<>();
-        HttpSession session = request.getSession();
-        String currentPage = (String) session.getAttribute(CURRENT_PAGE);
+        Map<String, String> map = new HashMap<>();
```
Comment (IlyaMatsuk) [inline src/main/java/by/matsuk/shop/controller/command/impl/admin/UpdateProductCommand.java]: please rename `map`

**Review by IlyaMatsuk — CHANGES_REQUESTED** [review]

File: src/main/java/by/matsuk/shop/controller/command/impl/admin/GoToUpdateProductPageCommand.java (hunk 1 above)
Comment (IraMatsuk) [reply src/main/java/by/matsuk/shop/controller/command/impl/admin/GoToUpdateProductPageCommand.java]: fixed

File: src/main/java/by/matsuk/shop/controller/command/impl/admin/UpdateProductCommand.java (hunk 3 above)
Comment (IraMatsuk) [reply src/main/java/by/matsuk/shop/controller/command/impl/admin/UpdateProductCommand.java]: fixed

File: src/main/java/by/matsuk/shop/controller/command/impl/admin/InsertNewProductCommand.java (hunk 2 above)
Comment (IraMatsuk) [reply src/main/java/by/matsuk/shop/controller/command/impl/admin/InsertNewProductCommand.java]: fixed

**Pull request merged**